t/e2e/whisper/whisper_mailbox_test.go

```diff
@@ -158,14 +168,22 @@ func (s *WhisperMailboxSuite) TestRequestMessagesInGroupChat() {
 
 	err = aliceBackend.StatusNode().AddPeer(mailboxEnode)
 	s.Require().NoError(err)
+	waitErr := helpers.WaitForPeerAsync(aliceBackend.StatusNode().GethNode().Server(), mailboxEnode, p2p.PeerEventTypeAdd, time.Second)
```

Comment:
Any reason why we don't group all the `PeerEventTypeAdd` checks after adding all the peers? That would seem closer to real-world behavior.